File: 58-length-of-last-word/length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        eachWordCount = []
        cnt = 0

        for i in range(0, len(s)):
            if i is not " ":
                start = i
                break
    
        for i in range(start, len(s)):
            if s[i] is " ":
                if cnt != 0:
                    eachWordCount.append(cnt)
                    cnt = 0
            else:
                cnt += 1
        
        if cnt != 0:
            eachWordCount.append(cnt)

        print(eachWordCount)
        return eachWordCount[-1]

File: 58-length-of-last-word/length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int right = s.size()-1;
        int cnt = 0;
        while(right >= 0 && s[right] == ' ') right--;
        while(right >= 0 && s[right] != ' ') {cnt++; right--;}
        return cnt;
    }
};